wgsl/index.bs

```diff
@@ -120,9 +120,9 @@ This specification does not prescribe a particular encoding for that text.
 
 ## Comments ## {#comments}
 
-Comments begin with `//` and continue to the end of the current line. There are no multi-line comments.
-
-TODO: What indicates the end of a line?  (E.g. A line ends at the next linefeed or at the end of the program)
+Line-comments begin with `//` and continue to the end of the current line or file.
+Block-comments begin with `/*` and continue to the end of the first subsequent `*/`. (no nesting)
```

Comment:
Why do you want block comments?
I think they are a source of confusion and pain.